knowledge-base/how-to-render-any-html-tags-in-reports.md

```diff
@@ -0,0 +1,81 @@
+---
+title: How to render any HTML tags and CSS attributes in reports
+description: How to render any HTML tags and CSS attributes in reports.
+type: how-to
+page_title: How to render any HTML tags and CSS attributes in reports
+slug: how-to-render-any-html-tags-in-reports
+res_type: kb
+---
+
+## Description
+
+A frequently asked question is *how to render HTML tags and CSS attributes that are not supported by [HtmlTextBox item](../report-items-html-text-box)* in reports.  
```

Comment:
Add more details,like why is this needed and used. Give some concrete scenarios and use cases.